zanata-war/src/main/java/org/zanata/action/ProjectAction.java

```diff
@@ -60,12 +60,10 @@ public DataModel getProjectPagedListDataModel() {
     }
 
     public boolean isShowObsolete() {
-        return showObsolete;
+        return false;
     }
 
     public void setShowObsolete(boolean showObsolete) {
-        projectPagedListDataModel.setFilterObsolete(!showObsolete);
```

Comment:
The reason I keep these methods is IDE refactoring doesn't seem to cover JSF pages. I will have to search all reference to projectAction.showObsolete in *.xhtml or pages.xml. Can be done but riskier. I will see what I can do.